packages/maskbook/src/plugins/Wallet/settings.ts

```diff
@@ -120,4 +120,6 @@ export const currentGasNowSettings = createGlobalSettings<GasNow | null>(
     (a: GasNow | null, b: GasNow | null) => isEqual(a, b),
 )
 
-connectGasNow()
+try {
+    connectGasNow()
+} catch {}
```

Comment:
plz note why need ignore catch

Reply:
It runs on the top level and it is the critical path of our program.

Reply:
```suggestion
} catch {
    // it runs on the top level and it is the critical path of our program.
}
```

Reply:
Actually, the correct fix is to move it elsewhere. I'll draft an issue for it.